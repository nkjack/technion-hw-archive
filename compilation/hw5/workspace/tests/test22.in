//var, var to var
void main() {
	byte a = 55b;
	byte c = a;
	a = 2b;
	printi(a);
	print("\n");
	printi(c);
	print("\n");
	
	int d = 101;
	int e = d;
	d = 1051;
	printi(d);
	print("\n");
	printi(e);
	print("\n");
	
	bool f = true;
	bool g = f;
	f = false;
	if (f) print("true"); else print("false");
	if (g) print("true"); else print("false");
	
	printi(a);
	printi(c);
	printi(d);
	printi(e);
	print("\n");
}