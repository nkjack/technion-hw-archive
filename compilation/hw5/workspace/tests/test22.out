Loaded: ./exceptions.s
2
55
1051
101
falsetrue2551051101
